portfolio/src/main/webapp/script.js

```diff
@@ -104,11 +104,21 @@ async function getHello() {
 
 //for parse json test
 async function getComments() {
-    //b99 quotes but from servlet instead of just js.
-    console.log('Fetching data json.');
-    const response = await fetch('/data');
-    console.log('Parsing json:');
-    const comments = await response.json();
+    console.log('checking if logged in:');
+    //set response to var login, then get json from it and set that to login
+    const login = await fetch('/loginstat').then(login => login.json());
+    console.log(login); //output to console json object
+    if(login.loggedIn){ //check if log in comparison works
+        console.log('logged-in');
+        document.getElementById('comment-form').innerHTML = '<form action=\"/data\" method=\"POST\"><textarea name=\"comment-input\">one, two, three</textarea><br/><br/><input type=\"submit\"/></form><p>Logout <a href=\"' + login.url + '\">here</a>.</p>';
```

Comment:
Another cleaner way to do this is to already have both of these elements in your HTML file (this way you have most if not all of your HTML code separated out in your HTML file). And then have a CSS class on both of them to hide them.


And then in this JS code, you can simply add CSS styling (or better yet, simply add/remove CSS classes) to make the one you want visible.